### DISCUSSION TITLE
Library.bib file has stopped updating and can't find Better BibTeX settings in Zotero 7

### DISCUSSION THREAD

**Discussion by jakobpoff** [retorquere/zotero-better-bibtex#2834] [Q&A]
I use Better BibTeX for Zotero integration with Obsidian. As of two days ago, my library.bib file has randomly stopped updating and in trying to fix the issue, I've realised I can't find the Better BibTeX settings panel within Zotero preferences. I hadn't tried to access these since upgrading to Zotero 7. The panels in my Zotero preferences are General, Sync, Export, Cite and Advanced. I've tried uninstalling and reinstalling Better BibTeX and have restarted Zotero to not success. Would be very grateful for help, I may be missing something obvious!

**Comment by retorquere** [comment]
Bug diagnosis is done in issues, not in discussions, but if you're on Zotero 7 you are likely experiencing #2829.
